buildpack/telemetry/mx_java_agent.py

```diff
@@ -128,10 +128,19 @@ def _enable_mx_java_agent(m2ee):
         m2ee, "-javaagent:{}{}".format(jar, mx_agent_args_str)
     )
 
-    # If not explicitly set, default to StatsD
+    # If not explicitly set,
+    # - default to StatsD (MxVersion < 9.7)
```

Comment:
change hardcoded 9.7 here too if we make it a variable

Reply:
Fixed